vkcs/kubernetes/data_source_cluster.go

```diff
@@ -138,6 +138,7 @@ func DataSourceKubernetesCluster() *schema.Resource {
 			"ingress_floating_ip": {
 				Type:        schema.TypeString,
 				Computed:    true,
+				Deprecated:  "This argument is no longer supported by vk cloud",
```

Comment:
We are VK cloud, there are no reason to add these "by's". :) Btw, it would be nice to add a bit info about how it should be done now.

Reply:
You forgot to update it here.

Reply:
fix